File: ass56/Program1.java
/*
1.Write a Java program which accept string from user and count number of capital characters. 
Input :   “Marvellous Multi OS” 
Output :   4
*/
import java.lang.*;
import java.util.*;
class ArrayDemo
{
    void Even(String str)
    {
        int iSum1 = 0,iPos = 0,iCnt = 0;
        for(int i=0; i<str.length(); i++)
        {
            char ch = str.charAt(i);
            if((ch>='A')&&(ch<='Z'))
            {
               iCnt++;
            }   
        }
        System.out.println("index is : " +iCnt);
    }
}
class Program1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Element");
        String str = sobj.nextLine();
        ArrayDemo dobj = new ArrayDemo();
        dobj.Even(str);
    }
}

File: ass27/program1.c
/*
1.Write a program which accept string from user and count number of capital characters. 
Input :   “Marvellous Multi OS” 
Output :   4 
*/
#include<stdio.h>
#include<string.h>
int Display(char str[])
{
    int i=0,iCnt=0;
    
    while(str[i]!='\0')
    {
        if((str[i]>=65)&&(str[i]<=90))
        {
        iCnt++;
      
        }
       i++;  
    }
    
   printf("%d",iCnt); 
}
int main()
{
    char arr[20];
    int iRet=0;

    printf("Enter The string\n");
    scanf("%[^'\n']s",arr);

    iRet=Display(arr);
    
    return 0;
}